harness/determined/pytorch/_callback.py

```diff
@@ -92,7 +92,8 @@ def on_checkpoint_end(self, checkpoint_dir: str) -> None:
         Deprecated. Please use :meth:`on_checkpoint_write_end` instead.
 
         .. warning::
-            This callback only executes on the chief GPU when doing distributed training.
+            This callback only executes on the chief GPU when doing distributed training with
```

Comment:
ln 17+18 need to be updated too.  seems like we no longer only call `on_validation_step_end` on chief for pytorchtrial.

Reply:
Cool.  I went ahead and updated all references to PyTorchTrial, actually.